plexupdate.sh

```diff
@@ -548,7 +548,7 @@ if [ ! -z "${PLEXSERVER}" -a "${AUTOINSTALL}" = "yes" ]; then
 fi
 
 if [ "${AUTOINSTALL}" = "yes" ]; then
-	if ! hash ldconfig 2>/dev/null && [ -x /sbin/ldconfig -a "${DISTRO}" = "ubuntu" ]; then
+	if ! hash ldconfig 2>/dev/null && [ -a "${DISTRO}" = "ubuntu" ]; then
```

Comment:
I'm still not understanding why we would limit this to ubuntu? Also, the -a isn't needed